src/convertToObject.js

```diff
@@ -1,12 +1,38 @@
 'use strict';
 
+const complexStylesString = require('./stylesString');
+
 /**
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
   // write your code here
+  const splited = sourceString.split(';');
```

Comment:
The variable name `splited` is a typo. It should be `split` to correctly represent the past tense of 'split'.